pom.xml

```diff
@@ -56,4 +78,16 @@
             <email>janarioliver@gmail.com</email>
         </developer>
     </developers>
+
+    <build>
+        <pluginManagement>
```

Comment:
@KostyaSha It is to use the maven-hpi-plugin, I was told it would give me some built-in tests. What is a the correct way to use maven-hpi-plugin?